[2024-09-10T08:19:06.603+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T08:19:06.698+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V5.third_task scheduled__2023-05-22T00:00:00+00:00 [queued]>
[2024-09-10T08:19:06.749+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V5.third_task scheduled__2023-05-22T00:00:00+00:00 [queued]>
[2024-09-10T08:19:06.753+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T08:19:06.868+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): third_task> on 2023-05-22 00:00:00+00:00
[2024-09-10T08:19:06.889+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=36528) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T08:19:06.901+0000] {standard_task_runner.py:72} INFO - Started process 36549 to run task
[2024-09-10T08:19:06.887+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V5', 'third_task', 'scheduled__2023-05-22T00:00:00+00:00', '--job-id', '8387', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp499ilqaw']
[2024-09-10T08:19:06.907+0000] {standard_task_runner.py:105} INFO - Job 8387: Subtask third_task
[2024-09-10T08:19:07.119+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V5.third_task scheduled__2023-05-22T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T08:19:07.375+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2023-05-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-05-22T00:00:00+00:00'
[2024-09-10T08:19:07.388+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T08:19:07.450+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T08:19:07.452+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the third task!']
[2024-09-10T08:19:07.470+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T08:19:07.473+0000] {subprocess.py:93} INFO - hey, this is the third task!
[2024-09-10T08:19:07.475+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T08:19:07.557+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T08:19:07.565+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V5, task_id=third_task, run_id=scheduled__2023-05-22T00:00:00+00:00, execution_date=20230522T000000, start_date=20240910T081906, end_date=20240910T081907
[2024-09-10T08:19:07.811+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T08:19:07.871+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T08:19:07.908+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
